src/components/form.module.scss

```diff
@@ -0,0 +1,165 @@
+@import "../mixins/button";
+
+.givingForm h1,
+.givingForm h2,
+.givingForm h3 {
+    font-family: var(--body-font);
+}
+
+.givingForm h3 {
+    text-transform: none;
+    font-weight: 400;
+}
+
+.datePicker {
+    font-family: inherit;
+}
+
+.formItemInput {
+    grid-area: input;
+    width: 100%;
+}
+
+.formItemMultipleChoiceChoices {
+    display: grid;
+    grid-area: choices;
+    grid-template-columns: 1fr;
+    grid-row-gap: var(--general-padding);
+}
+
+.accountDetails {
+    display: grid;
+    grid-column-gap: var(--general-padding);
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-areas: "label label label info info info info info . . . .";
+    margin-bottom: var(--general-padding);
+}
+
+@media only screen and (max-width: 968px) {
+    .accountDetails {
+        grid-template-columns: repeat(6, 1fr);
+        grid-template-areas: "label label label info info info";
+    }
+}
+
+.accountDetailLabel {
+    grid-area: label;
+}
+
+.accountDetailInfo {
+    grid-area: info;
+    font-style: italic;
+}
+
+.sectionInfo {
+    margin-bottom: var(--general-padding);
+}
+
+.declaration {
+    font-style: italic;
+    margin-bottom: var(--general-padding);
+}
+
+select, .dateInput {
```

Comment:
so that this rule is more specific, could you give the `<select>` elements the class, `styles.select` and have this say `.select, .dateInput { ...`? Otherwise, this rule will apply to all `<select>` elements on the site